go-controller/pkg/libovsdb/ops/model.go

```diff
@@ -489,15 +490,73 @@ func buildFailOnDuplicateOps(c client.Client, m model.Model) ([]ovsdb.Operation,
 	return c.WhereAny(m, cond).Wait(ovsdb.WaitConditionNotEqual, &timeout, m, field)
 }
 
-// getAllUpdatableFields returns a list of all of the columns/fields that can be updated for a model
-func getAllUpdatableFields(model model.Model) []interface{} {
+// ModelUpdateField enumeration represents fields that can be updated on the supported models
+type ModelUpdateField int
+
+const (
+	LogicalSwitchPortAddresses ModelUpdateField = iota
+	LogicalSwitchPortType
+	LogicalSwitchPortTagRequest
+	LogicalSwitchPortOptions
+	LogicalSwitchPortPortSecurity
+	LogicalSwitchPortEnabled
+
+	PortGroupACLs
+	PortGroupPorts
+	PortGroupExternalIDs
+)
+
+// getUpdatableFieldInterfaces gets a model and a list of ModelUpdateField and returns a list of their related interface{} fields.
+func getUpdatableFieldInterfaces(model model.Model, fieldNames []ModelUpdateField) []interface{} {
```

Comment:
nit: `getFieldsToUpdate` or something like that

Reply:
DONE